R/splitByRanks.R

```diff
@@ -121,37 +116,53 @@ setGeneric("splitByRanks",
     SimpleList(ans)
 }
 
-#' @rdname splitByRanks
+#' @rdname agglomerate-methods
 #' @export
-setMethod("splitByRanks", signature = c(x = "SummarizedExperiment"),
-    function(x, ranks = taxonomyRanks(x), na.rm = TRUE, ...){
+setMethod("agglomerateByRanks", signature = c(x = "SummarizedExperiment"),
+    function(x, ranks = taxonomyRanks(x), na.rm = TRUE, as.list = FALSE, ...){
+        browser()
```

Comment:
browser()

--> Have you run tests? They should notify you about this. (devtools::check() or BiocCheck)